uint8 BOOT_STATUS_OFF=1
uint8 BOOT_STATUS_PINGING=2
uint8 BOOT_STATUS_PING_SUCCESSFUL=3
uint8 BOOT_STATUS_WAITING_FOR_LOG=4
uint8 BOOT_STATUS_GOT_LOG=5
uint8 BOOT_STATUS_WAITING_FOR_BOOT_SUCCESS=6
uint8 BOOT_STATUS_BOOT_SUCCESS=7
uint8 BOOT_STATUS_BOOT_FAILED=8
uint8 BOOT_STATUS_BOOT_FAIL_NO_ARM=9
uint8 BOOT_STATUS_BOOT_FAIL_NO_RELAY=10
uint8 BOOT_STATUS_BOOT_CANCELED=11

uint8 boot_status

